packages/inter-protocol/src/provisionPool.js

```diff
@@ -16,7 +16,7 @@ import { prepareProvisionPoolKit } from './provisionPoolKit.js';
 
 /** @import {Marshal} from '@endo/marshal'; */
 
-/** @type {ContractMeta} */
+/** @type {ContractMeta<typeof start>} */
 export const meta = {
```

Comment:
Aside: Is the upgrade of zcf that knows to look for `privateArgsShape` under `meta` deployed to mainnet?

@Chris-Hibbert is working on upgrading the provisionPool in an upcoming release.